Sources/Swifter.swift

```diff
@@ -114,6 +115,14 @@ public class Swifter {
             NotificationCenter.default.removeObserver(token)
         }
     }
+
+    private var storedSession: Any?
+    @available(macOS 10.15, *)
+    @available(iOS 13.0, *)
+    internal var session: ASWebAuthenticationSession? {
+        get { return storedSession as? ASWebAuthenticationSession }
```

Comment:
Why are you storing this separately as an Any?

Reply:
Otherwise, I need to put `@available` annotation to the Swifter class.

![Screen Shot 2020-12-08 at 10 39 30](https://user-images.githubusercontent.com/19896354/101427256-e5715780-3941-11eb-96b0-3f2cfa7bc910.png)

Reply:
And, `stored properties cannot be marked potentially unavailable with '@available'`.

![Screen Shot 2020-12-08 at 10 42 27](https://user-images.githubusercontent.com/19896354/101428390-f58a3680-3943-11eb-8e41-ac5ede965d5b.png)

Reply:
If we don't set `session` as the stored property of the Swifter class, the alert will disappear immediately after it is displayed, and the user will not be able to continue the procedure.

![alert](https://user-images.githubusercontent.com/19896354/101503589-995ffa80-39b5-11eb-99cf-1554ad0b6677.png)